source/PlayServicesResolver/src/PlayServicesResolver.cs

```diff
@@ -641,19 +641,34 @@ public class AndroidSdkRootChangedArgs : EventArgs {
                 AndroidSdkRootChanged += ResolveOnAndroidSdkRootChange;
                 ScheduleAutoResolve();
             }
-
-            // Register events to monitor build system changes for the Android Resolver and other
-            // plugins.
-            RunOnMainThread.OnUpdate += PollBundleId;
-            RunOnMainThread.OnUpdate += PollBuildSystem;
-            RunOnMainThread.OnUpdate += PollAndroidAbis;
-            RunOnMainThread.OnUpdate += PollAndroidSdkRoot;
+			
+			if (GooglePlayServices.SettingsDialog.EnableAutoResolution)
+				LinkAutoResolution();
         }
 
-        /// <summary>
-        /// Called from PlayServicesSupport to log a message.
-        /// </summary>
-        internal static void LogDelegate(string message, PlayServicesSupport.LogLevel level) {
+		public static void UnlinkAutoResolution() {
+			// Unregister events to monitor build system changes for the Android Resolver and other
+			// plugins.
+			RunOnMainThread.OnUpdate -= PollBundleId;
```

Comment:
In general this all looks good.  However, Firebase and some other plugins rely upon the bundle ID notification to keep other configuration files up to date.  So turning off auto-resolution should probably not turn off bundle ID polling.  Any chance we could keep this unconditionally enabled?

Reply:
From what I observed, the really heavy hitter was the polling for the android ABIs, so long as that doesn't run all the time the bulk majority of the allocations should go away. I'd trust your judgement on this one regarding polling the bundleid, I just removed any events that hooked into editor update tick. Not really sure how I can reformat my commit message, is that possible with Git?

Reply:
Yeah you can reformat your commit message with "git commit --amend", save a new commit message and force push the change to the head of your review repo.